primer-android/res/values/strings.xml

```diff
@@ -254,13 +254,13 @@
     <string name="btc_donate">Bitcoin Donate:</string>
 
     <!-- mode -->
-    <string name="choose_mode_cold_desctription">Primecoin Cold : Offline &amp; Secure.</string>
-    <string name="choose_mode_warm_desctription">Primecoin Hot : Online &amp; Simple.</string>
+    <string name="choose_mode_cold_desctription">Primer Cold : Offline &amp; Secure.</string>
+    <string name="choose_mode_warm_desctription">Primer Hot : Online &amp; Simple.</string>
     <string name="choose_mode_warm_waiting">Loading…</string>
     <string name="choose_mode_warm_retry">Retry</string>
-    <string name="choose_mode_warm_change_to_cold">Switch to Primecoin Cold</string>
-    <string name="choose_mode_cold_confirm">Once mode is set, it can\'t be changed.\nPrimecoin Cold need to be run on a offline phone.\nSet this phone as Primecoin Cold?</string>
-    <string name="choose_mode_warm_confirm">Once mode is set, it can\'t be changed.\nPrimecoin Hot is an online bitcoin wallet.\nSet this phone as Primecoin Hot?</string>
+    <string name="choose_mode_warm_change_to_cold">Switch to Primer Cold</string>
+    <string name="choose_mode_cold_confirm">Once mode is set, it can\'t be changed.\nPrimer Cold need to be run on a offline phone.\nSet this phone as Primer Cold?</string>
+    <string name="choose_mode_warm_confirm">Once mode is set, it can\'t be changed.\nPrimer Hot is an online bitcoin wallet.\nSet this phone as Primer Hot?</string>
```

Comment:
here it should be a 'online primecoin wallet' rather than 'online bitcoin wallet'